tests/integration/Settings_API/AbstractSettingsTest.php

```diff
@@ -72,6 +72,27 @@ public function test_register_setting() {
 	}
 
 
+	/**
+	 * @see Abstract_Settings::load_settings()
+	 *
+	 * Stored values are defined in get_settings_instance().
+	 */
+	public function test_load_settings() {
+
+		$this->get_settings_instance()->register_setting( 'test-setting-d', Setting::TYPE_EMAIL, [
+			'name'        => 'Test Setting D',
+			'description' => 'Description of setting D',
+		] );
+
+		// TODO: uncomment assert for test-setting-b when https://github.com/skyverge/wc-plugin-framework/pull/453 is merged {WV 2020-03-20}
```

Comment:
This can be uncommented now.

Reply:
Good catch @deivamagalhaes. I just committed the suggested changes and fixed the merge conflict with the base branch.